Translation corrections (#477)
07th-mod/higurashi-console-arcs: Update/omot_021.txt

			NULL, "\"Just in case something were to happen, if you will.", Line_WaitForInput);
	ModPlayVoiceLS(4, 3, "ps3/s14/03/220300114", 256, TRUE);
	OutputLine(NULL, "期待しすぎるとよくないけど、調べてみる価値はあると思うよ」",

> 			NULL, " It wouldn't be good to expect much on that end, but I do think it's worth investigating.\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " It's best not to expect much on that end, but I do think it's worth investigating.\"", Line_Normal);
```
